authentication-service/src/main/kotlin/com/saveourtool/save/authservice/utils/SaveUserDetails.kt

```diff
@@ -45,8 +49,14 @@ class SaveUserDetails(
         httpHeaders.set(AUTHORIZATION_ID, id.toString())
         httpHeaders.set(AUTHORIZATION_NAME, name)
         httpHeaders.set(AUTHORIZATION_ROLES, role)
+        httpHeaders.set(AUTHORIZATION_STATUS, status)
     }
 
+    /**
+     * @return true if [status] is [UserStatus.ACTIVE], false otherwise
+     */
+    fun isActive() = status == UserStatus.ACTIVE.toString()
```

Comment:
suggestion to change implementation for `isEnabled` to check status

Reply:
Will done for now, bit later it seems that we would support several user statuses but not only `ACTIVE`

Reply:
Done